docs/upgrade/11_to_12/workflows_mariadb.py

```diff
@@ -164,9 +166,9 @@ def parse_bool(value):
   `continuable` tinyint(1) DEFAULT 0,
   `completed` datetime DEFAULT NULL,
   `started` datetime DEFAULT NULL,
-  `description` varchar(255) DEFAULT NULL,
+  `description` longtext DEFAULT NULL,
   `exception_handler_workflow` varchar(255) DEFAULT NULL,
-  `if_condition` varchar(255) DEFAULT NULL,
+  `if_condition` varchar(65535) DEFAULT NULL,
```

Comment:
This caused:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/mysql/connector/connection_cext.py", line 535, in cmd_query
    self._cmysql.query(query,
_mysql_connector.MySQLInterfaceError: Column length too big for column 'if_condition' (max = 16383); use BLOB or TEXT instead
```

After setting `varchar(16383)` I got

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/mysql/connector/connection_cext.py", line 535, in cmd_query
    self._cmysql.query(query,
_mysql_connector.MySQLInterfaceError: Row size too large. The maximum row size for the used table type, not counting BLOBs, is 65535. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs
```

so I used `varchar(2555)`